__tests__/app.test.js

```diff
@@ -25,3 +24,29 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of all topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(topics.length).toBe(3);
+        topics.forEach((topic) => {
+          expect(typeof topic.description).toBe("string");
+          expect(typeof topic.slug).toBe("string");
+        });
+      });
+  });
+});
+
+describe("general error tests", () => {
+  test("404: error message when given an invalid endpoint", () => {
+    return request(app)
+      .get("/api/oh-noes")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("Not found");
+      });
+  });
+});
```

Comment:
Good job with this error test, it belongs in it's own describe block and the path is distinctive!